data/systemd/system/yggdrasil.service.in

```diff
@@ -18,3 +18,4 @@ CacheDirectory=yggdrasil
 
 [Install]
 WantedBy=multi-user.target
+Alias=yggd.service
```

Comment:
The RHEL-9 contains `rhcd.server`. What distribution and version contains `yggd.service`?

Reply:
Upstream yggdrasil ships the unit as `yggd.service`, so I followed the pattern of upstream yggdrasil setting an alias for its old/deprecated unit name. I feel like we should keep the `rhc` package in RHEL responsible for the `rhcd.service` unit name, even if it is just a symlink (or even a drop-in file that adds an `Alias=` directive to `yggdrasil.service`).

Reply:
Thanks